jdaviz/configs/imviz/plugins/parsers.py

```diff
@@ -261,6 +284,27 @@ def _jwst2data(file_obj, ext, data_label):
 
     return data, new_data_label
 
+# ---- Functions that handle input from Roman ASDF files -----
+
+
+def _roman_2d_asdf_to_glue_data(file_obj, data_label):
+    ext_list = ['data', 'dq', 'err', 'var_poisson', 'var_rnoise']
+
+    for ext in ext_list:
+        comp_label = ext.lower()
+        new_data_label = f'{data_label}[{comp_label}]'
+        data = Data(label=new_data_label)
+
+        # For now, set bunit to a blank string.
+        bunit = ''
```

Comment:
Will roman datamodels ship the unit info in the future? Seems weird to ship the data without unit.